src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Utilities/AsyncSymbolVisitor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal abstract class AsyncSymbolVisitor : SymbolVisitor<ValueTask>
+    {
+    }
```

Comment:
is this type valuable enough to have?

Reply:
Not sure. It seemed weird to have one but not the other.